### PR TITLE
Allow user to pass in name or wikipedia link

### PR DESCRIPTION
**Pull request by bnmajor**
If a 'name' or 'wikipedia' field are passed in via the body of a molecule those fields will be set. 

If 'name' or 'wikipedia' are included in the input parameters of a calculation and the molecule does not yet exist those fields will be set upon molecule creation. If the molecule already exists the fields are ignored.

### CODE REVIEW

**Comment by bnmajor** [comment]
Works with openchemistry/mongochemclient#166 to display the wikipedia link.

File: girder/molecules/molecules/utilities/molecules.py [hunk 1]
```diff
@@ -80,11 +80,18 @@ def create_molecule(data_str, input_format, user, public, gen3d=True,
...
+        if 'name' in parameters:
+            name = parameters['name']
+        else:
+            name = chemspider.find_common_name(inchikey)
         if name is not None:
             mol_dict['name'] = name
 
+        # Set a wikipedia link if one is provided
+        if 'wikipedia' in parameters:
+            mol_dict['wiki'] = parameters['wikipedia']
```
Comment (cjh1) [inline girder/molecules/molecules/utilities/molecules.py]: Lets call this `wikipediaUrl'

Comment (bnmajor) [reply girder/molecules/molecules/utilities/molecules.py]: Rename just the input parameters key(`wikipedia`), the molecule key(`wiki`), or both?

Comment (cjh1) [reply girder/molecules/molecules/utilities/molecules.py]: I would say both.

Comment (bnmajor) [reply girder/molecules/molecules/utilities/molecules.py]: Fixed for both. Also updated in openchemistry/mongochemclient#166

**Pull request merged**